tools/wptrunner/wptrunner/stability.py

```diff
@@ -91,9 +107,29 @@ def is_inconsistent(results_dict, iterations):
     return len(results_dict) > 1 or sum(results_dict.values()) != iterations
 
 
+def find_slow_status(test):
+    """Check if a single test almost times out.
+
+    We are interested in tests that almost time out (i.e. likely to be flaky).
+    Therefore, timeout statuses are ignored, including (EXTERNAL-)TIMEOUT &
+    CRASH (tests that time out may be marked as CRASH if crashes are detected).
+
+    Returns:
+        A result status produced by a run that almost times out; None, if no
+        runs almost time out.
+    """
+    threshold = test["timeout"] * FLAKY_THRESHOLD
+    for status in ['PASS', 'FAIL', 'OK', 'ERROR', 'INTERNAL-ERROR']:
+        if (status in test["longest_duration"] and
+                test["longest_duration"][status] > threshold):
```

Comment:
Weird indentation (yeah I know pyflakes likes it, but I think it's a bad rule which we don't enforce).